deploy/build_install_deploy.sh

```diff
@@ -3,11 +3,22 @@
 # Exit on any error
 set -e
 
+# Color codes
+GREEN="\033[42;30m"
+RED="\033[41;30m"
+RESET="\033[0m"
+
 # Define required variables with no defaults
 BUILD_TYPE=""
 NUM_KEEPERS=""
 NUM_GRAPHERS=""
 
+# Set default values for optional flags
+DEPLOY=false
+STOP=false
+RESET=false
```

Comment:
this collides with the color code reset.

Reply:
I suggest we have  DEPLOY, START, STOP, CLEAN .